packages/odata-v4/src/index.ts

```diff
@@ -16,28 +13,29 @@ export {
   contains
 } from './filter-functions';
 
-export { ActionImportRequestBuilder } from './request-builder/action-import-request-builder';
-export { ActionImportParameter } from './request/action-import-parameter';
-export { ODataBatchRequestBuilder } from './batch-request-builder';
-
-export { CustomField } from './selectable/custom-field';
-export { edmToTs } from './payload-value-converter';
-export { FunctionImportRequestBuilder } from './request-builder/function-import-request-builder';
-
 export {
   transformReturnValueForComplexType,
   transformReturnValueForComplexTypeList,
   transformReturnValueForEdmType,
   transformReturnValueForEdmTypeList,
   transformReturnValueForEntity,
   transformReturnValueForEntityList,
-  transformReturnValueForUndefined
-} from './request-builder/response-transformers';
-export { deserializeComplexType } from './entity-deserializer';
+  transformReturnValueForUndefined,
+  CreateRequestBuilder,
+  DeleteRequestBuilder,
+  GetAllRequestBuilder,
+  GetByKeyRequestBuilder,
+  UpdateRequestBuilder,
+  ActionImportRequestBuilder,
+  FunctionImportRequestBuilder
+} from './request-builder';
```

Comment:
```suggestion
  transformReturnValueForUndefined
} from './request-builder/response-transformers';

export { CreateRequestBuilder } from './request-builder/create-request-builder';
export { DeleteRequestBuilder } from './request-builder/delete-request-builder';
export { GetAllRequestBuilder } from './request-builder/get-all-request-builder';
```
export { GetByKeyRequestBuilder } from './request-builder/get-by-key-request-builder';
export { UpdateRequestBuilder } from './request-builder/update-request-builder';